driver/main.ml

```diff
@@ -55,15 +55,15 @@ let main () =
            [make_archive;make_package;compile_only;output_c_object])
         > 1
     then begin
-      let module P = Clflags.Compiler_pass in
       match !stop_after with
       | None ->
         fatal "Please specify at most one of -pack, -a, -c, -output-obj";
-      | Some (P.Parsing | P.Typing) ->
+      | Some _ ->
```

Comment:
One last nitpick, it is better to keep an exhaustive list of `Clflags.Compiler_flags` to remind the reader of all possible interactions.